app/tests/Core/documentTests.js

```diff
@@ -1686,6 +1695,10 @@ export default [
               text: 'My note',
             },
           ],
+          id: {
+            system_name: 'My system',
+            text: 'My system test',
```

Comment:
Just to make the usage clear => link to example from spec.
```suggestion
            system_name: 'GitHub Issue',
            text: 'oasis-tcs/csaf#210',